test/revert.spec.ts

```diff
@@ -4489,7 +4687,7 @@ describe(`Reverts (Seaport v${VERSION})`, function () {
               value,
             }
           )
-      ).to.be.revertedWith(`NOT_AUTHORIZED`);
+      ).to.be.revertedWith("NOT_AUTHORIZED");
```

Comment:
`.revertedWith` now checks that the return data is an encoded `Error(string)`, and performs an exact match with the reason string in the revert.